<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CENTAUR_NAMESPACE::NetworkAddress</class>
 <widget class="QDialog" name="CENTAUR_NAMESPACE::NetworkAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>363</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border: 0px;
    background-color: rgb(25, 25, 25);
    border-radius: 5px;
    min-height: 25px;
    padding: 0 0 0 10;
}

QLineEdit:hover {
    background-color: rgb(35, 35, 35);
}

QLineEdit:focus {
    background-color: rgb(45, 45, 45);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalInfoLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="image">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="coinName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>64</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="btnCopyTag">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Copy Tag to clipboard</string>
       </property>
       <property name="whatsThis">
        <string>Copy to clipboard</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: black;
border: 0px;
border-radius: 5px;
background-color: rgb(68, 138, 255);
}

QPushButton:hover, QPushButton:hover:default{
	background-color:rgb(83,147,255);
}
QPushButton:pressed{
background-color: rgb(93,157,255);
}
QPushButton:default, QPushButton:pressed:default{
background-color: rgb(93,157,255);
}
QPushButton:disabled{
color: #c0c0c0;
background-color: #9D9EA4;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrc/resource.qrc">
         <normaloff>:/img/copy</normaloff>:/img/copy</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelURL">
       <property name="text">
        <string notr="true">URL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditTag">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnCopyAddress">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Copy address to clipboard</string>
       </property>
       <property name="whatsThis">
        <string>Copy to clipboard</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: black;
border: 0px;
border-radius: 5px;
background-color: rgb(68, 138, 255);
}

QPushButton:hover, QPushButton:hover:default{
	background-color:rgb(83,147,255);
}
QPushButton:pressed{
background-color: rgb(93,157,255);
}
QPushButton:default, QPushButton:pressed:default{
background-color: rgb(93,157,255);
}
QPushButton:disabled{
color: #c0c0c0;
background-color: #9D9EA4;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrc/resource.qrc">
         <normaloff>:/img/copy</normaloff>:/img/copy</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditURL">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAddress">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnCopyURL">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Copy URL to clipboard</string>
       </property>
       <property name="whatsThis">
        <string>Copy to clipboard</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: black;
border: 0px;
border-radius: 5px;
background-color: rgb(68, 138, 255);
}

QPushButton:hover, QPushButton:hover:default{
	background-color:rgb(83,147,255);
}
QPushButton:pressed{
background-color: rgb(93,157,255);
}
QPushButton:default, QPushButton:pressed:default{
background-color: rgb(93,157,255);
}
QPushButton:disabled{
color: #c0c0c0;
background-color: #9D9EA4;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrc/resource.qrc">
         <normaloff>:/img/copy</normaloff>:/img/copy</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTag">
       <property name="text">
        <string>Tag</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelNetwork">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Network</string>
       </property>
       <property name="buddy">
        <cstring>networks</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="networks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
